include/swift/Strings.h

```diff
@@ -21,15 +21,6 @@ constexpr static const char SERIALIZED_MODULE_EXTENSION[] = "swiftmodule";
 constexpr static const char SERIALIZED_MODULE_DOC_EXTENSION[] = "swiftdoc";
 /// The extension for PCH files.
 constexpr static const char PCH_EXTENSION[] = "pch";
-/// The extension for replacement map files.
-constexpr static const char REMAP_EXTENSION[] = "remap";
-/// The extension for SIL files.
-constexpr static const char SIL_EXTENSION[] = "sil";
-/// The extension for SIB files.
-constexpr static const char SIB_EXTENSION[] = "sib";
-/// The extension for LLVM IR files.
-constexpr static const char LLVM_BC_EXTENSION[] = "bc";
-constexpr static const char LLVM_IR_EXTENSION[] = "ll";
```

Comment:
I have an okay reason. :-/ The remaining constants here are ones that are used in libraries below Frontend (in layering terms). I'd prefer not to put them *all* here because I think the `file_types` abstractions are better, but it does make an ugly asymmetry here.

Another option would be to go a step further and sink the information in `file_types` down to Basic, which means they could be included from anywhere.

Reply:
I *like* that option: sinking the info down to Basic! It's a nice unification which I suspect will pay off in the future!